runtime/vm/ContinuationHelpers.cpp

```diff
@@ -309,47 +322,16 @@ walkAllStackFrames(J9VMThread *currentThread, J9StackWalkState *walkState)
 		localWalkState.walkThread = targetThread;
 		rc = vm->walkStackFrames(currentThread, &localWalkState);
 
-		if (NULL != targetThread->currentContinuation) {
-			localWalkState = *walkState;
-			walkContinuationStackFrames(currentThread, targetThread->currentContinuation, &localWalkState);
-		}
 		targetThread = targetThread->linkNext;
 	} while (targetThread != vm->mainThread);
 
-	/* Walk all contination stacks */
-	if (NULL != vm->liveVirtualThreadList) {
-		j9object_t rootNode = J9_JNI_UNWRAP_REFERENCE(vm->liveVirtualThreadList);
-		j9object_t nextVThread = J9OBJECT_OBJECT_LOAD(currentThread, rootNode, vm->virtualThreadLinkNextOffset);
-		J9VMContinuation *continuation = NULL;
-		while (nextVThread != rootNode) {
-			j9object_t cont = J9VMJAVALANGVIRTUALTHREAD_CONT(currentThread, nextVThread);
-			continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(currentThread, cont);
-			if (NULL != continuation) {
-				/* Reset localWalkState */
-				localWalkState = *walkState;
-				/* walk live continuation's stack */
-				rc = walkContinuationStackFrames(currentThread, continuation, &localWalkState);
-			}
-			nextVThread = J9OBJECT_OBJECT_LOAD(currentThread, nextVThread, vm->virtualThreadLinkNextOffset);
-		}
-	}
-	/* Alternate approach to rely on GC Continuation object iterator
-		GC_ContinuationIterator iter = getContinuationIterator();
-		J9VMContinuation *continuation = NULL;
-		while(NULL != iter) {
-			continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(currentThread, nextCont);
-			if (NULL != continuation) {
-				// Reset localWalkState
-				localWalkState = *walkState;
-				// walk live continuation's stack
-				rc = walkContinuationStackFrames(currentThread, continuation, &localWalkState);
-				if (J9_STACKWALK_RC_NONE != rc) {
-					goto exit;
-				}
-			}
-			iter = iter.next();
-		}
-	 */
+	/* Walk all live continuation stacks using the GC Continuation object iterator */
+	PORT_ACCESS_FROM_VMC(currentThread);
+	vm->memoryManagerFunctions->j9mm_iterate_all_continuation_objects(currentThread,
+																privatePortLibrary,
```

Comment:
the first parameter should be on a new line: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line
```suggestion
	vm->memoryManagerFunctions->j9mm_iterate_all_continuation_objects(
																currentThread,
																privatePortLibrary,
```